src/fstests/fstests.jinja2

```diff
@@ -36,6 +36,16 @@ class Job(BaseJob):
         except Exception as e:
             print('Kernel build error:', e)
 
+    def _get_xml_from_vm(self, kdir):
+        try:
+            result = subprocess.run([f"debugfs -R \"dump ext4/results-4k/results.xml {kdir}/results.xml\" /home/kernelci/alex/fstests-poc/fstests/disks/vdg"], check=True, shell=True)
```

Comment:
@gctucker I got what you meant about kernel source tree for results, but in the case of VM files, do you have any other suggestion?

Reply:
`/home/kernelci/alex/fstests-poc/fstests/disks/vdg`

Surely we can use a relative path for that?  Also, I don't see what you mean.  Can you not copy the xml file to `kdir` which is the kernel source tree?

Reply:
As discussed, one option is to add a command line argument e.g. `--xfstests-bld-path` and store it in the Job class, then use it in the string here e.g. `f"{self._xfstests_bld_path}/run-fstests/disks/vdg"`.

Reply:
Or maybe a template variable and then it will be defined in the script globals, like the version in `kver.jinja2`.

Reply:
Updated to use `--xfstests-bld-path`

Reply:
I don't see this, the path is still hard-coded to `/home/kernelci/alex` in this commit.